README.md

```diff
@@ -1,87 +1,81 @@
+The SymPy Live shell is a thin wrapper around the `repl` app in the JupyterLite.
+In order to reduce the maintentance needs,
+we leverage the code from the [`jupyterlite/demo`](https://github.com/jupyterlite/demo) project,
+and provide only minimal customization: a custom landing page that loads the REPL in an iframe.
 
-See Developer instructions below for installing the prerequisites.
 
-Run jupyter lite locally:
-```bash
-jupyter lite serve
-```
+### Configs
 
-Build ready for distribution via static serve:
-
-```bash
-jupyter lite build
-```
-
-The files will be saved in `_output/`.
+The operation of JupyterLite is configured in several places:
+- `requirements.txt`: specifies the version `jupyterlab`, `jupyterlite-core` and `jupyterlite-pyodide-kernel`.
+- `jupyter_lite_config.json`: general configs for the jupyter lite build system
+- `repl/jupyter-lite.json`: configs specific to the REPL app. See [example here](https://github.com/ivanistheone/live/blob/357e60a228b43ac28ef835953d00f4495a429d78/repl/jupyter-lite.json).
+- `overrides.json`: customizations of the JupyterLite UI
 
+In addition to the above, the SymPy Live shell depends on the following customizations:
+- Custom landing page [index.html](https://github.com/sympy/live/blob/main/templates/index.html) that embeds the JupyterLite REPL.
+- Automatic import `from sympy import *` and init commands in [index.html template](https://github.com/sympy/live/blob/main/templates/index.html#L6-L11).
+- JavaScript code to decode evaluate query string, see [here](https://github.com/sympy/live/blob/main/templates/index.html#L49-L56).
 
-You can test the files using any static server, for example:
+> [!NOTE]
+> Updating the version of SymPy running on [live.sympy.org/](https://live.sympy.org/) usually requires updating the version of `jupyterlab`, `jupyterlite-core`, and `jupyterlite-pyodide-kernel` listed in `requirements.txt`. Use the [upstream requirements file](https://github.com/jupyterlite/demo/blob/main/requirements.txt) as reference. See [here](https://pyodide.org/en/stable/usage/packages-in-pyodide.html) for the backage verison included in Pyodide. 
 
-```bash
-cd _output/
-python -m http.server
-```
 
 
-## Configs
+### Deploying in production
 
-The operation of JupyterLite can be configured in several places:
-- `jupyter_lite_config.json`: general configs for the jupyter lite build system
-- `repl/jupyter-lite.json`: configs specific to the REPL app. See [example here](https://github.com/ivanistheone/live/blob/357e60a228b43ac28ef835953d00f4495a429d78/repl/jupyter-lite.json).
-- `overrides.json`: customizations of the JupyterLite UI
-- `requirements.txt`: Python packages needed to build this project (used by CI)
+SymPy Live Shell is hosted as a static website on GitHub Pages with a custom domain.
+Deployment to production is automated by the Github actions workflow [deploy.yml](https://github.com/sympy/live/blob/main/.github/workflows/deploy.yml),
+which performs the following steps:
```

Comment:
It would be good to mention here that deployment happens any time a pull request is merged to the main branch.

Reply:
I say this a little bit lower in the instrucitons.